aviary/subsystems/mass/gasp_based/fixed.py

```diff
@@ -39,7 +42,7 @@ def setup(self):
         num_engine_type = len(self.options['aviary_options'].get_val(
             Aircraft.Engine.NUM_ENGINES))
 
-        add_aviary_input(self, Aircraft.Wing.SWEEP, val=25)
+        add_aviary_input(self, Aircraft.Wing.SWEEP, val=25, units='deg')
```

Comment:
If we change this to:
```suggestion
        add_aviary_input(self, Aircraft.Wing.SWEEP, val=0.436, units='rad')
```
we can remove deg_to_rad from the calculations

Reply:
Do you mean that we use 'rad' in both `design_load.py` and `fixed.py`?

Reply:
yes

Reply:
Now, I understand.

Reply:
done